ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/ContextExtensions.kt

```diff
@@ -0,0 +1,21 @@
+package com.revenuecat.purchases.ui.revenuecatui
+
+import android.app.Activity
+import android.content.Context
+import android.content.ContextWrapper
+
+/**
+ * This function gets the activity from a given context. Most times, the context itself will be
```

Comment:
Super nit, but I recommend not starting docs with stuff like "this function...". That's implied since it's document it 😅
```suggestion
 * Returns the activity from a given context. Most times, the context itself will be
```